src/kudu/tserver/tablet_server_options.h

```diff
@@ -36,6 +41,8 @@ struct TabletServerOptions : public kudu::server::ServerBaseOptions {
 
   std::vector<HostPort> tserver_addresses;
 
+  std::shared_ptr<kudu::log::LogFactory> log_factory;
```

Comment:
I thought `tablet_server_options` was for primitive setup variables for tserver.